pom.xml

```diff
@@ -161,5 +161,11 @@
             <artifactId>PistonUtils</artifactId>
             <version>1.3.2</version>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.11.0</version>
+        </dependency>
+
```

Comment:
First of all the newline after this text is not needed, and secondly it has to be relocated.